src/app/auth-guard.service.ts

```diff
@@ -1,23 +0,0 @@
-import { AuthService } from './auth.service';
-import { Injectable } from '@angular/core';
-import { CanActivate, Router, RouterStateSnapshot } from '@angular/router';
-import { map } from 'rxjs/operators'
-
-@Injectable({
-  providedIn: 'root'
-})
-export class AuthGuard implements CanActivate {
-
-  constructor(private auth: AuthService, private router: Router) { }
-
-  canActivate(router, state: RouterStateSnapshot) {
-    const user = this.auth.user$.pipe(map(user => {
```

Comment:
`user` might mean an object with properties (user.name, user.token,..). So a better variable name would be `isLoggedIn` or `isUser`